src/docs/cookbook-manage/implement-delete-user/index.md

```diff
@@ -0,0 +1,671 @@
+---
+title: Implement "Delete profile" button
+slug: implement-delete-user
+updated: 2022-03-07
+category: cookbook-manage
+ingress:
+  This article guides you to implement a feature where a user can delete
+  their own profile through the FTW template Profile page. TODO - update
+  "published"!
+published: false
```

Comment:
If you change this to  `published: true`, then this article can be previewed through the preview link